app/jinja_filters.py

```diff
@@ -2,24 +2,44 @@
 
 import re
 import string
+from decimal import InvalidOperation
 
 from datetime import datetime
 
 import flask
 
 from jinja2 import Markup, escape, evalcontextfilter
 
-from babel import units
+from babel import units, numbers
 
 blueprint = flask.Blueprint('filters', __name__)
 
 
 @blueprint.app_template_filter()
-def format_currency(value):
-    if value is not None and len(str(value)) > 0:
-        return "£{:,.2f}".format(value)
-    else:
-        return ""
+def get_formatted_number(value=None, decimal_places=0):
+    formatted_number = ''
+    if value:
+        try:
+            formatted_number = numbers.format_number(value, locale='en_GB')
+            if decimal_places and decimal_places > 0:
+                if '.' not in formatted_number:
+                    formatted_number += '.'
+                while len(formatted_number.split('.')[1]) < decimal_places:
+                    formatted_number += '0'
+        except InvalidOperation:
+            return value
+
+    return formatted_number
+
+
+@blueprint.app_template_filter()
+def get_currency_symbol(currency="GBP"):
+    return numbers.get_currency_symbol(currency, locale='en_GB')
+
+
+@blueprint.app_template_filter()
+def format_currency(value=None, currency="GBP"):
+    return numbers.format_currency(number=value, currency=currency, locale='en_GB')
```

Comment:
Given we're using the `en_GB` locale in a few places it would we better to define it as a constant, something like `DEFAULT_LOCALE='en_GB'`